packages/react/src/components/Masthead/MastheadTopNav.js

```diff
@@ -21,36 +25,62 @@ import {
  * @returns {*} Masthead top nav component
  */
 const MastheadTopNav = ({ navigation, ...topNavProps }) => {
+  /**
+   * Event data for CoreMetrics
+   *
+   * @param {string} title title of the menu item
+   * @param {number} key key for id of menu item
+   * @param {string} type type of element
+   * @returns {object} event fields
+   */
+  const menuItemEventData = (title, key, type) => ({
+    type: 'element',
+    primaryCategory: 'MASTHEAD',
+    eventName: 'HOVER',
+    executionPath: `masthead__l0-nav--nav-${key}`,
+    execPathReturnCode: type,
+    targetTitle: title,
+  });
+
   /**
    * Top masthead navigation
    *
    * @returns {*} Top masthead navigation
    */
-
   const mastheadLinks = navigation.map((link, i) => {
     if (link.hasMenu) {
       return (
-        <HeaderMenu
-          aria-label={link.title}
-          menuLinkName={link.title}
-          data-autoid={`masthead__l0-nav--nav-${i}`}>
-          {link.menuSections[0].menuItems[
-            i
-          ].megapanelContent.quickLinks.links.map((item, j) => {
-            return (
-              <HeaderMenuItem
-                href={item.url}
-                data-autoid={`masthead__l0-nav--subnav-${j}`}>
-                {item.title}
-              </HeaderMenuItem>
-            );
-          })}
-        </HeaderMenu>
+        <div
+          className={`${prefix}--masthead__nav-item`}
+          onMouseEnter={() =>
+            analytics(menuItemEventData(link.title, i, 'dropdown'))
+          }>
+          <HeaderMenu
+            onMouseEnter={() => analytics(menuItemEventData(link.title, i))}
```

Comment:
Is there a reason to have two `onMouseEnter` right after each other here like this?

Reply:
whoops yep - removing that one